solutions/community-deploy/CORTX-Deployment.md

```diff
@@ -0,0 +1,84 @@
+# Setup K8s Cluster and Deploy CORTX Stack
+
+   Please find below steps to setup K8s cluster and deploy CORTX Stack.
+
+## Prerequisite 
+### Minimum specification required for Cortx Stack
+   - RAM: 16GB
+   - CPU: 8Core
+   - DISK: Need 8 Disk with 10GB per disk.
+
+### If you have automated deployment for solution yaml, please make sure your vm having below drives available
```

Comment:
Did not understand this. `automated deployment for solution.yaml`? do we have any  automated deployment job for community?

Reply:
@AbhijitPatil1992 , what you mean by above sentence?

Reply:
I have added note in a script for deployment steps for solution yaml file.

Reply:
is it like ` if you want automated deployment with solution.yaml` ?

Reply:
It's like if you want to create solution.yaml file automatically then we need to give automated value for this else if you want to pass your solution yaml file then need to give manual value for this variable.

Reply:
then it should be something like `If you set SOLUTION_CONFIG_TYPE as automated to generate solution.yaml automatically, please make sure your vm having below drives available`